packages/grafana-data/src/types/config.ts

```diff
@@ -237,7 +237,7 @@ export interface GrafanaConfig {
   listScopesEndpoint?: string;
   reportingStaticContext?: Record<string, string>;
   exploreDefaultTimeOffset?: string;
-  exploreDisableLogsDownload?: boolean;
+  ExploreHideLogsDownload?: boolean;
```

Comment:
It shouldn't be capitalized here, right?

Reply:
Good catch! Fixed in 83de5b674d64fea1dd251f7772ff75756a04d05b